src/app/layout/qgslayoutdesignerdialog.cpp

```diff
@@ -108,6 +108,8 @@
 #include <ApplicationServices/ApplicationServices.h>
 #endif
 
+#include "gdal.h"
```

Comment:
```suggestion
#include <gdal.h>
```

System include?

Reply:
changed to that. (it looks we have a mix within the code base of both styles). I believe the difference between both styles isn't fully standardized and compilers might or might not implement different behaviors